tests/Factory/Simple/SimpleSentryFactoryTest.php

```diff
@@ -105,29 +105,35 @@ public function testSameSentryInstance(): void
 		Assert::assertSame($sentry, $factory->getSentry(new SentryIdentificator('string')));
 	}
 
-	public function testNoSentry(): void
+	/**
+	 * @return mixed[][]|\Generator
+	 */
+	public function getSentryWhenNoSentryForIdentificatorDataProvider(): Generator
 	{
-		$factory = new SimpleSentryFactory(new SentryIdentificatorParser());
-		$sentryIdentificator = new SentryIdentificator('');
-
-		try {
-			$factory->getSentry($sentryIdentificator);
-			Assert::fail('Exception expected');
-		} catch (\Consistence\Sentry\Factory\NoSentryForIdentificatorException $e) {
-			Assert::assertSame($sentryIdentificator, $e->getSentryIdentificator());
-		}
+		yield 'empty string' => [
+			'sentryIdentificatorWithInvalidId' => new SentryIdentificator(''),
+		];
+		yield 'nonexistent object' => [
+			'sentryIdentificatorWithInvalidId' => new SentryIdentificator('Foo\Bar'),
+		];
 	}
 
-	public function testNonexistingObject(): void
+	/**
+	 * @dataProvider getSentryWhenNoSentryForIdentificatorDataProvider
+	 *
+	 * @param \Consistence\Sentry\Metadata\SentryIdentificator $sentryIdentificatorWithInvalidId
+	 */
+	public function testGetSentryWhenNoSentryForIdentificator(
+		SentryIdentificator $sentryIdentificatorWithInvalidId
```

Comment:
```suggestion
		SentryIdentificator $sentryIdentificator
```

I think it is not really "invalid" and not all properties/identificators have sentries.

Reply:
Renamed.